CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 2015-08-18 Release 1.2.0
+
+### Backwards-incompatible changes
+
+None
+
+### Summary
+
+Enhanced support for Amazon Linux
+Ability to set HOME environment variable when installing npm
+Misc lint/spec fixes
```

Comment:
These three points should have two under `Added` and one under `Fixed` instead of being under `Summary` if you want to follow the style of http://keepachangelog.com/